docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -309,9 +309,6 @@ This archive collects the following information:
 
 Note that the diagnostics bundle is intended for debugging purposes only, its structure may change between releases.
 
-IMPORTANT: The configuration files in the archive may have credentials in *plain text*.
-These will need to be manually redacted before the archive is shared.
```

Comment:
> IMPORTANT: {agent} attempts to automatically redact credentials and API keys when creating diagnostics. Please review the contents of the archive before sharing to ensure there are no credentials in plain text.

We have an algorithm for automatically redacting credentials, but it is best effort and may miss things. Rather than remove this I would rather document that we attempt to do the redaction but not to rely on it completely. For example, there have been bugs in this algorithm before.